src/layout/Summary2/config.tsx

```diff
@@ -0,0 +1,67 @@
+import { CG, Variant } from 'src/codegen/CG';
+import { CHECKBOX_SUMMARY_PROPS } from 'src/layout/Checkboxes/config';
+import { CompCategory } from 'src/layout/common';
+import { DROPDOWN_SUMMARY_PROPS } from 'src/layout/Dropdown/config';
+import { INPUT_SUMMARY_PROPS } from 'src/layout/Input/config';
+import { MULTIPLE_SELECT_SUMMARY_PROPS } from 'src/layout/MultipleSelect/config';
+import { RADIO_SUMMARY_PROPS } from 'src/layout/RadioButtons/config';
+export const Config = new CG.component({
+  category: CompCategory.Container,
+  rendersWithLabel: false,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+  },
+})
+  .addProperty(
+    new CG.prop(
+      'children',
+      new CG.arr(new CG.str())
+        .optional()
+        .setTitle('Children')
+        .setDescription('Array of component IDs that should be displayed in the summary'),
+    ).onlyIn(Variant.External),
+  )
+  .addProperty(new CG.prop('childComponents', new CG.arr(CG.layoutNode)).onlyIn(Variant.Internal))
+  .addProperty(
+    new CG.prop(
+      'children',
+      new CG.arr(new CG.str())
+        .optional()
+        .setTitle('Children')
+        .setDescription('Array of component IDs that should be displayed in the summary'),
+    ).onlyIn(Variant.External),
+  )
+  .addProperty(
+    new CG.prop(
+      'whatToRender',
+      new CG.obj(
+        new CG.prop(
+          'type',
+          new CG.enum('page', 'layoutSet', 'component', 'task')
+            .optional({ default: 'component' })
+            .setTitle('Mode')
+            .setDescription('Config for what should be rendered'),
+        ),
+        new CG.prop('id', new CG.str()),
+      ),
+    ),
+  )
+  .addProperty(
+    new CG.prop(
+      'overWriteProperties',
```

Comment:
How about just calling this `overrides`? This property is overly long and complex for something we expect app developers to use a lot - and it's not effectively overwriting anything. Also, they're all properties, thats implicit - but by using 'properties' in the name it may look like you can override any property in the component configuration (which you can't).

Reply:
Agree :)